internal/reconciler/shoot/add.go

```diff
@@ -78,7 +80,61 @@ func (r *Reconciler) IsRelevantShoot(obj client.Object) bool {
 	return true
 }
 
-// IsRelevantShootUpdate checks whether the old or new object is a relevant Shoot
+// IsRelevantShootUpdate triggers reconciliation for the following cases:
+// A Shoot becoming relevant or irrelevant using IsRelevantShoot.
+// The service-account-issuer changed.
+// Shoot is marked for deletion.
```

Comment:
This would make the doc string formatted in an easier to read way

<img width="782" height="217" alt="Image" src="https://github.com/user-attachments/assets/3c6db5f5-1b91-467a-a987-fc6213f0289f" />

Also the `[]` are used to link to the function `IsRelevantShoot` https://tip.golang.org/doc/comment#links

```suggestion
// IsRelevantShootUpdate triggers reconciliation for the following cases:
//   - a Shoot becoming relevant or irrelevant using [IsRelevantShoot]
//   - the service-account-issuer changed
//   - a Shoot being marked for deletion
```